app/templates/eventView.html

```diff
@@ -106,7 +106,7 @@ <h3>Program Trainings</h3>
               {% if (not isPastEvent) and (not isNewEvent)%}
                   {% if userHasRSVPed %}
                     <button type="submit" class="btn btn-danger" value="{{eventData.id}}" id="removeRsvpBtn">Remove RSVP</button>
-                  {%else%}
+                  {% elif (not isPastEvent) and (eventData.isRsvpRequired) %}
```

Comment:
we know the first part will always be True, so we can leave it out of the expression